setup.py

```diff
@@ -69,21 +69,23 @@ def get_ucp_version():
     "pynvml",
 ]
 
-tests_require = [
-    "pytest",
-    "pytest-asyncio",
-]
+extras_require = {
+    "test": [
+        "pytest",
+        "pytest-asyncio",
+    ]
+}
 
 setup(
-    name="ucx-py",
+    name="ucx-py"+os.getenv("RAPIDS_PY_WHEEL_CUDA_SUFFIX", default=''),
```

Comment:
Wonder if we could capture this another way. Perhaps like a common dependency that all packages depends on?